tests/baselines/test_psr_discovery.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import numpy as np
+
+from grl import load_spec, MDP, AbstractMDP
+from grl.baselines.psr_jong import discover_tests
+
+def test_discovery_tiger():
+    """
+    From Littman et al. (2003): We should discover 2 core tests for Tiger.
+    """
+    spec = load_spec('tiger')
+    mdp = MDP(spec['T'], spec['R'], spec['p0'], spec['gamma'])
+    amdp = AbstractMDP(mdp, spec['phi'])
+
+    # assert conditions are the same as in Littman et al. (2003)
+    assert amdp.n_obs == 2, f"Number of observations is {amdp.n_obs}"
+    assert amdp.n_actions == 3, f"Number of actions is {amdp.n_actions}"
```

Comment:
These assertions currently fail here and in paint - why are the obs/action spaces different?